packages/ui/package.json

```diff
@@ -19,6 +19,7 @@
     "virtual-dom-stringify": "^3.0.1",
     "vue": "3.2.26",
     "vue-router": "4.0.12",
+    "vue3-click-away": "^1.2.1",
```

Comment:
We already have `@vueuse/core` which has [onClickOutside](https://vueuse.org/core/onClickOutside)